API.md

```diff
@@ -111,7 +111,61 @@ Adds comment with the attachment from the URL to a particular task.
 
 #### Arguments
 
-- `taskId` ID of the task to which the comment should be added
+- `taskId` - ID of the task to which the comment should be added
 - `commentText` - Text to add as a comment
 - `attachmentUrl` - URL of the attachment file
 - `attachmentFileName` - Name under which the attachment should be saved
+
+## "Escape hatch" methods
+
+We don't document every single operation possible, but you still can pass an undocumented extra object without reverting to `fetch()` once this basic API wrapper is not sufficient. For advanced applies.
+
+### `client.createByPojo(collectionName, pojo)`
```

Comment:
eh, I don't like POJO, let's just call it `createRaw(collectionName, rawRecord)` (consistent naming with WatermelonDB)

Reply:
Cool, I'll rename it.